framework/packages/abstract-core/src/objects/module.rs

```diff
@@ -476,9 +476,13 @@ impl Default for Monetization {
 /// Module Metadata String
 pub type ModuleMetadata = String;
 
+pub const SALT_POSTFIX: &[u8] = b"abstract";
 /// Generate salt helper
 pub fn generate_instantiate_salt(account_id: &AccountId) -> Binary {
-    let hash = <sha2::Sha256 as sha2::Digest>::digest(account_id.to_string());
+    let account_id_hash = <sha2::Sha256 as sha2::Digest>::digest(account_id.to_string());
+    let mut hash = account_id_hash.to_vec();
+    hash.extend(SALT_POSTFIX);
```

Comment:
Could we add a comment on why this is needed?

Reply:
Depends if we keep it, haha

Reply:
Don't think adding our organization name in salt needs explanation tho

Reply:
can salt be arbitrary length?

Reply:
up to 64 bytes, and we're using only 40 right now. 32 for sha256(account_id) and 8 for b"abstract".
We have checks for length:
https://github.com/AbstractSDK/abstract/blob/7ea32f60664d6a56eb4ef509c5a3af6439a37370/framework/packages/abstract-core/src/objects/module.rs#L833-L838